model_compression_toolkit/core/common/substitutions/batchnorm_folding.py

```diff
@@ -182,6 +183,7 @@ def __init__(self,
                                                       with the batch normalization weights
             get_kernel_hw_fn: Function for getting the kernel height & width shape
             is_group_conv_fn: Function for checking if the linear layer is a group-convolution
+            is_bn_node_valid_fn: Function for checking whether the node to forward fold is a valid node and it's type
```

Comment:
Not critical: consider another name: "get_bn_node_type_fn" which returns "bn", "dw" and None...